src/index.js

```diff
@@ -1,28 +1,79 @@
-import React from 'react';
+import React, { Component } from 'react';
 import ReactDOM from 'react-dom';
+import createHistory from 'history/createBrowserHistory'
 import {
   BrowserRouter as Router,
-  Route
+  Route,
+  matchPath
 } from 'react-router-dom'
 
-import App from './App';
+import Game from './components/Game';
 import Landing from './components/Landing';
 import './index.css';
 
-ReactDOM.render(
-  <Router>
-    <div className='app'>
-      <h1>Album Hangman</h1>
-      <div className='container'>
-        <Route exact path={'/'} component={Landing} />
-        <Route path={'/game/:username/:period?/:hideArtwork?'} component={App} />
+const GAME_PATH = '/game/:username?/:period?/:hideArtwork?';
+
+class App extends Component{
```

Comment:
Can you explain why you decided to put App here instead of a separate file?

Reply:
The only reason was to state the changes while an `App.js` was present at that time. 
I agree that this should now move out to a separate file.

Reply:
Let's do it in this PR then :-)